ui/app/AppLayouts/Wallet/panels/WalletHeader.qml

```diff
@@ -60,12 +60,12 @@ Item {
 
         StatusAddressPanel {
             objectName: "addressPanel"
-            address: currentAccount.mixedcaseAddress
-
+            value: currentAccount.ens || currentAccount.mixedcaseAddress
+            ens: !!currentAccount.ens
```

Comment:
At first glance I expect `ens` property to store the ENS name, however this is a bolean that triggers displaying of ENS name instead of address. What do you think of a comment to show its scope or maybe `showEns` as the property name? will this clarify its purpose better?